docs_src/sidebar/sidebar_data.py

```diff
@@ -47,7 +47,7 @@
                 'vision.gan': '/vision.gan',
                 'vision.model overview': '/vision.models',
                 'vision.models.unet': '/vision.models.unet',
-                'vision.widgets': '/widgets.image_cleaner'
+                'widgets.image_cleaner': '/widgets.image_cleaner'
```

Comment:
@fpingham, I changed the namespace for widgets in the sidebar to reflect where they actually are, seems like a mismatch in master.

Reply:
Yes, this is correct.